typings/globalTypes.d.ts

```diff
@@ -2,6 +2,16 @@ type Constructor<T, TA = any> = new(...args: TA[]) => T;
 type ExtendTypeWith<T extends Constructor<any>, OtherObject extends object> = Constructor<InstanceType<T> & OtherObject, ConstructorParameters<T>>;
 type Dictionary<TYPE> = {[key: string]: TYPE};
 
+declare module 'react-native-color' {
```

Comment:
Not sure where non-typed js dependencies are declared, so I've added these dependencies here. Is this the right place?